Justfile

```diff
@@ -138,6 +138,37 @@ start-oracle oracle-name trigger-oracle-build-type="--use-local-cargo-artifacts"
 build-blocksense:
   @{{root-dir}}/scripts/build-blocksense.sh
 
+[group('General')]
+[doc('Evaluate a Nix expression for a specific machine')]
+eval-machine machine commit="working-tree":
+  #!/usr/bin/env bash
+  set -euo pipefail
+  if [[ "{{ commit }}" == "working-tree" ]]; then
+    NIX_TARGET="{{ root-dir }}"
+  else
+    NIX_TARGET="github:blocksense-network/blocksense?rev={{commit}}"
+  fi
+  nix eval --raw "${NIX_TARGET}#nixosConfigurations.{{machine}}.config.system.build.toplevel.outPath"
+
+[group('General')]
+[doc('List NixOS integration tests')]
+list-nixos-tests:
+  #!/usr/bin/env bash
+  set -euo pipefail
+  nix eval --json .#legacyPackages.x86_64-linux.nixosTests --apply "builtins.attrNames" | \
+    jq -r '.[]'
+
+[group('General')]
+[doc('Run NixOS integration tests')]
+run-nixos-tests:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    tests=$(just list-nixos-tests)
+    for test in $tests; do
+      echo "Running NixOS test: $test"
+      nix build --option sandbox false -L --json --accept-flake-config .#legacyPackages.x86_64-linux.nixosTests."$test"
```

Comment:
Why the disabled sandbox?

Reply:
Most likely to enable internet access, needed for both `anvil --fork-url` and the oracle scripts.